payments-core/src/main/java/com/stripe/android/model/PaymentMethodCreateParams.kt

```diff
@@ -36,6 +36,7 @@ data class PaymentMethodCreateParams internal constructor(
     private val cashAppPay: CashAppPay? = null,
     private val swish: Swish? = null,
     val billingDetails: PaymentMethod.BillingDetails? = null,
+    private val allowRedisplay: PaymentMethod.AllowRedisplay? = null,
```

Comment:
iOS is currently setting this as `unspecified` by default since `null` and `unspecified` have the same behavior (API sets to `unspecified` when null). Should we do the same?